package.json

```diff
@@ -19,19 +19,24 @@
     "organization"
   ],
   "scripts": {
-    "offline-lint:js": "eslint .",
-    "fix": "eslint --fix .",
-    "start": "node ./dist/bin/www",
-    "start-in-container": "node ./bin/www",
+    "build": "tsc",
```

Comment:
Larger diff here since I sorted the scripts, and dropped the `./node_modules/typescript/bin/` from the command

Reply:
k, thanks.
One of our devs regularly hits issues in some environments where `tsc` doesn't work. I forget if it's a Windows-only or a Mac-only example they had. We'll see how it goes!

Reply:
This should be cross platform, since it's invoked through NPM, and it will pick the right script in BIN. I think some platforms it's `.bin` vs `bin`